tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -310,3 +310,148 @@ user enters text into meta guidance data file content editor
     user clicks element  ${editor}
     user waits until page contains testid   fileGuidanceContent-focused
     user enters text into element  ${editor}  ${text}
+
+
+user adds summary text block to content section
+    [Arguments]  ${text}
+    user clicks button  Add a summary text block 
+    user clicks button  Edit block 
+    user enters text into element  //*[@role="textbox"]//p  ${text} 
+    user clicks button  Save
+
+
+user adds release note to content 
+    [Arguments]  ${text}
+    user clicks button  Add note
+    user enters text into element  //*[@id="createReleaseNoteForm-reason"]  ${text}
+
+user creates amendment for release
+    [Arguments]  ${PUBLICATION_NAME}  ${RELEASE_NAME}  ${RELEASE_STATUS}
+    user goes to url  %{ADMIN_URL}
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    user opens details dropdown   ${RELEASE_NAME} ${RELEASE_STATUS}  ${accordion}
+    ${details}=  user gets details content element  ${RELEASE_NAME} ${RELEASE_STATUS}  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    user clicks button  Amend this release  ${details}
+    user clicks button  Confirm
+
+user replaces data files for amendment 
+    [Arguments]  ${SUBJECT_NAME}  ${SUBJECT_FILE}  ${META_FILE}  ${text}  ${ERROR}
+    [Documentation]  EES-1442
+    user clicks link  Data and files
```

Comment:
I think I'd remove `user clicks link Data and files` from this keyword and put it in the test suite itself, so its clearer to people reading the tests what is going on.